Makefile

```diff
@@ -11,7 +11,7 @@ build:
 
 .PHONY: test
 test:
-	@RUST_BACKTRACE=1 cargo unit-test
+	@RUST_BACKTRACE=1 cargo unit-test --features=enable-test-utils
```

Comment:
Without this flag, the shared test code won't compile, so it's a requirement for all test runs.  Fortunately, with the `Makefile` abstraction, this doesn't change anything unless you're running tests directly from `cargo` instead of using the nice `make test` command